task4/src/models/index.js

```diff
@@ -1 +1,24 @@
-export { default as UserModel } from './user';
+import fs from 'fs';
+import path from 'path';
+
+import sequelize from '../data-access';
+
+const db = {};
+
+// eslint-disable-next-line no-sync
+fs.readdirSync(__dirname)
+    .filter((file) => file.indexOf('.') !== 0 && file !== 'index.js')
+    .forEach((file) => {
+        const model = require(path.join(__dirname, file)).default;
+        db[model.name] = model.init(sequelize);
+    });
+
+Object.keys(db).forEach((modelName) => {
+    if (typeof db[modelName].associate === 'function') {
+        db[modelName].associate(db);
+    }
+});
+
+sequelize.sync();
+
```

Comment:
What is this code for? If it's needed to sync data in DBs, then usually it's done with deployment scripts, not in terms of application

Reply:
It allows to create tables, if they aren't exist. And used to add {force: true} there, when I needed to add or modify columns in my tables.